mesh/mesh_initializer.py

```diff
@@ -0,0 +1,40 @@
+__doc__ = """ Mesh Initializer using Pyvista """
+
+import pyvista as pv
+import numpy as np
+
+
+class MeshInitialize():
+    """
+    This Mesh Initializer class uses pyvista to import mesh files in the
+    STL or OBJ file formats and initializes the necessary mesh information.
+
+    How to initialize a mesh?
+    -------------------------
+
+    mesh = MeshInitialize(r('<filepath>'))
+
+    PS: Please be sure to add .stl / .obj at the end of the filepath, if already present, ignore.
+    """
+    def __init__(self, filepath):
+        self.mesh = pv.read(filepath)
+        self.pyvista_faces = self.mesh.faces
+        self.number_of_faces = self.mesh.n_faces
+        self.pyvista_points = self.mesh.points
+        self.faces = self.face_calculation(self.pyvista_faces, self.pyvista_points, self.number_of_faces)
+
+    def face_calculation(self, pvfaces, meshpoints, n_faces):
+        """
+        This function converts the faces from pyvista to pyelastica geometry
+        """
```

Comment:
@armantekinalp  I was thinking that here on wards we should try to make it a habit of adding type hints and including the information in doc string as needed. Ofc eventually we need to propagate this backward in `pyelastica` as well. Thoughts?

Reply:
I agree with you @bhosale2 . We will take care already published 'pyelastica' code base later but new code can have type hints.  @Rahul-JOON can you add type hints for the new code in this PR.

Reply:
Yes, sure I will add them.